simpleclient/src/main/java/io/prometheus/client/Counter.java

```diff
@@ -130,6 +135,12 @@ public void inc(double amt) {
     public double get() {
       return value.sum();
     }
+    /**
+     * Get the created time of the counter in milliseconds.
+     */
+    public long created() {
```

Comment:
Should this maybe be a float in seconds? That way callers don't need to divide by 1000.0 to get the value they should expose, also since this accessor doesn't have the unit (seconds/milliseconds/etc) maybe it's best to return the unit that's expected to be exposed for exposition?

Reply:
Java's current time function returns milliseconds, so that's the effective standard inside Java - and thus also what the client uses everywhere already.